R/activate.R

```diff
@@ -156,22 +156,27 @@ renv_activate_prompt <- function(action, library, prompt, project) {
   if (!ask)
     return(FALSE)
 
-  fmt <- lines(
-    "",
-    "This project has not yet been activated.",
-    "Activating this project will ensure the project library is used when %s() is called.",
-    "Please see `?renv::activate` for more details.",
-    ""
+  renv_activate_prompt_impl(project)
+
+
+}
+
+renv_activate_prompt_impl <- function(project = NULL) {
+  choices <- c(
+    "Activate the project, setting up a project library",
+    "Continue, using the system library",
+    "Cancel"
```

Comment:
Do you think this is necessary? Since `menu()` will tell you how to cancel:

```
> menu("Testing")

1: Testing

Selection: 
Enter an item from the menu, or 0 to exit
Selection: 0
[1] 0
```

although I guess it's not displayed when the menu is first displayed.

Reply:
(Is there prior precedent for this from other R packages?)

Reply:
```suggestion
    "Cancel."
```

Reply:
Yeah, I think the `0` option is pretty hard to discover. There's some related art in rsconnect, but that's because I solved a similar problem a few of weeks ago.